db/schema.rb

```diff
@@ -10,194 +10,187 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170330225217) do
+ActiveRecord::Schema.define(version: 20170610162757) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "hstore"
 
-  create_table "activities", force: :cascade do |t|
-    t.integer  "team_id"
-    t.string   "kind",         limit: 255
-    t.string   "guid",         limit: 255
-    t.string   "author",       limit: 255
-    t.string   "title",        limit: 255
-    t.text     "content"
-    t.string   "source_url",   limit: 255
+  create_table "activities", id: :serial, force: :cascade do |t|
```

Comment:
Ok, I knew about AR now using `bigint` for id's, but my own Rails 5.1 app doesn't add the type explicitly. 
So, does that mean that we now have to define a `:serial` explicitly when we create a migration? I just ask this because if so, it would be helpful if you can update the readme accordingly. 
Also: if people have an older version of pg running, will that be a problem?  The Readme says: '9.3' or higher.